packages/react-components/react-popover/library/src/components/Popover/Popover.types.ts

```diff
@@ -97,7 +97,8 @@ export type PopoverProps = Pick<PortalProps, 'mountNode'> & {
   closeOnIframeFocus?: boolean;
 
   /**
-   * Configures the position of the Popover
+   * Configures the position of the Popover.
+   * Explore [Positioning docs](https://react.fluentui.dev/?path=/docs/concepts-developer-positioning-components--docs) for more options.
```

Comment:
```suggestion
   * Explore [Positioning docs](?path=/docs/concepts-developer-positioning-components--docs) for more options.
```

See: https://github.com/microsoft/fluentui/blob/master/apps/public-docsite-v9/src/Concepts/StylingComponents.stories.mdx?plain=1#L118